src/core-sdk-tasks/BuildFPMToolPreReqs.cs

```diff
@@ -6,7 +6,6 @@
 using Microsoft.Build.Utilities;
 using Microsoft.Build.Framework;
 using System.IO;
```

Comment:
Since you're cleaning up some of the code, might as well sort the usings too

Reply:
So, I didn't run the same formatters/analyzers like I did in the SDK repo on this code. Here, I likely just did "Remove unused usings". However, the point is kinda moot when you look at this file in the SDK PR:
https://github.com/dotnet/sdk/pull/38804/files#diff-ea4fa261d2db3cc7e1e241b5f6ac3a003ff19fdde37c09ddb089b7467277160dR4

There's only 1 using because all of these are in the global usings of that repo. So... 🙃